infrastructure/smart-contracts/contracts/upgrades/referee/Referee9A.sol

```diff
@@ -253,41 +253,6 @@ contract Referee9A is Initializable, AccessControlEnumerableUpgradeable {
         return esXai(esXaiAddress).totalSupply() + Xai(xaiAddress).totalSupply() + _allocatedTokens;
     }
 
-    /**
-     * @notice Toggles the assertion checking.
-     */
-    function toggleAssertionChecking() external onlyRole(DEFAULT_ADMIN_ROLE) {
-        isCheckingAssertions = !isCheckingAssertions;
-        emit AssertionCheckingToggled(isCheckingAssertions);
-    }
-	
-    /**
-     * @notice Sets the challengerPublicKey.
-     * @param _challengerPublicKey The public key of the challenger.
-     */
-    function setChallengerPublicKey(bytes memory _challengerPublicKey) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        challengerPublicKey = _challengerPublicKey;
-        emit ChallengerPublicKeyChanged(_challengerPublicKey);
-    }
-
-    /**
-     * @notice Sets the rollupAddress.
-     * @param _rollupAddress The address of the rollup.
-     */
-    function setRollupAddress(address _rollupAddress) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        rollupAddress = _rollupAddress;
-        emit RollupAddressChanged(_rollupAddress);
-    }
```

Comment:
For these we have to remove / comment out the cli cmds that would call these functions.
We also need to make sure the CLI will build with the new Referee Abi

Reply:
Added Story for removing these

https://www.pivotaltracker.com/story/show/187906357